_posts/2018-10-16-cypress-vs-selenium.md

```diff
@@ -0,0 +1,261 @@
+---
+layout: post
+title:  "Move over Selenium - UI Testing with Cypress"
+date:   2018-10-16 02:00:00 -0800
+cover:  /assets/images/cypress/logo-dark.cy.png
+excerpt: Cypress is a relatively new front end testing tool that can be used for your UI testing 
```

Comment:
This needs to be updated so the excerpt is all on one line. Right now the build breaks because there is a line break after the word "testing". 

Sorry, there are no PR tests yet to ensure the build passes. Just noticed this when testing locally.

```sh
cjvmbp2k16:godaddy.github.io chrisvogt$ bundle exec jekyll serve
Configuration file: /Users/chrisvogt/Dev/godaddy.github.io/_config.yml
            Source: /Users/chrisvogt/Dev/godaddy.github.io
       Destination: /Users/chrisvogt/Dev/godaddy.github.io/_site
 Incremental build: disabled. Enable with --incremental
      Generating...
             Error: YAML Exception reading /Users/chrisvogt/Dev/godaddy.github.io/_posts/2018-10-16-cypress-vs-selenium.md: (<unknown>): could not find expected ':' while scanning a simple key at line 7 column 1
```